### ISSUE TITLE
Debug Accounts

### ISSUE THREAD

**Issue by MichailiK** [Avalon-Entertainment/bkrpg-support#26]
_Suggested by Michaili#0406, extended by Shockwave#2019_

Setting an Account as a "Debug Account" opens up the possibility to manipulate all Stats, HP/MP, Items in inventory etc. no matter what if they are in a battle or not for the purpose of verifying bugs. The Debug Account "Flag" must be set by the owner of the bot (@avalonrising) to a user and opens up a Debugging battle channel, where normal spawning can apply, but can be overridden by a Debug Account to let a certain enemy immediately appear.

TL;DR:
- Debug Accounts that @avalonrising can declare which ones are Debug Accounts
- Debug Accounts serve the purpose of verifying bugs & testing features
- Debug Accounts can manipulate anything on **their** character (HP, MP, Stats, Give & Take Items etc.) no matter what.
- A Debug battle channel will be opened for Debug Accounts. Normal spawns apply, but can be overridden to spawn a specific enemy / boss immediately.

**Comment by MichailiK** [comment]
Also, Debug Accounts won't be shown in any Leaderboards, and may apply Patreon perks to themselves.
